styles.css

```diff
@@ -728,3 +756,661 @@ a {
+    padding: 0;
+    gap: 24px;
+    width: 448px;
+    height: 348px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .multi_story {
+    width: 448px;
+    height: 48px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #091e42;
+    margin-top: -10px;
+  }
+
+  .see_prjt_buttn {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 10px;
+    width: 124px;
+    height: 48px;
+    background: #f55800;
+    flex: none;
+    order: 3;
+    flex-grow: 0;
+    margin-top: 5px;
+  }
+
+  .recent_work_text {
+    width: 448px;
+    height: 140px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #091e42;
+    margin-top: -5px;
+  }
+
+  .recent_project {
+    display: grid;
+    gap: 24px;
+    margin-top: 10rem;
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .card_bttn {
+    background: #ff6b00;
+    padding: 12px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    border: none;
+    color: white;
+    cursor: pointer;
+    width: 100%;
+    z-index: 10;
+    transition: all 0.5s ease-in-out;
+    opacity: 1;
+  }
+
+  .project_background {
+    position: relative;
+    background: url('image folder/projects-background.png');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 95%;
+    height: 395px;
+  }
+
+  .project_foreground {
+    position: relative;
+    background:
+      linear-gradient(
+        179.35deg,
+        rgba(38, 38, 38, 0) 0.85%,
+        rgba(38, 38, 38, 0.9) 84%
+      );
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    transition: transform 600ms cubic-bezier(0.73, 0.59, 0, 0.55);
+  }
+
+  .work_content {
+    display: flex;
+    flex-direction: column;
+    gap: 12px;
+    padding: 16px;
+    height: 386px;
+    width: 327px;
+    justify-content: flex-end;
+  }
+
+  .work_content h3 {
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+  }
+
+  .work_content p {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #fff;
+  }
+
+  .project_tech {
+    margin: 0 16px;
+  }
+
+  .project_tech ul {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-bottom: 22px;
+  }
+
+  .project_foreground:hover {
+    transform: scale(1.02);
+    transform-origin: 0 0;
+    z-index: 10000;
+    opacity: 1;
+    background: transparent;
+    cursor: pointer;
+  }
```

Comment:
- [ ] Nice work here, but this is still not the ideal outcome expected according to the Figma design, as stated by the previous reviewer, you need to make the buttons disappear when the hover event is not active and appear on hover, currently they are displayed in all cases.